crates/fluvio-connector-common/src/config.rs

```diff
@@ -13,6 +13,11 @@ pub fn value_from_file<P: Into<PathBuf>>(path: P) -> Result<Value> {
 }
 
 pub fn from_value<T: DeserializeOwned>(value: Value, root: Option<&str>) -> Result<T> {
```

Comment:
should have unit tests for these

Reply:
Added a test in https://github.com/infinyon/fluvio/pull/3138/commits/cda51fa1e06684af9fcc0ddf2d7c0b622d0225d5